### PR TITLE
Automatic commit: Move 'nwbc-cflp-integration' from QA to Production

### PR DESCRIPTION
**Pull request by sandra36**
my first tutorial - appreciate your feedback!

### CODE REVIEW

File: tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+parser: v2
+auto_validation: true
+time: 20
+tags: [ tutorial>beginner, software-product-function>ui-sap-business-client--nwbc-]
+primary_tag: software-product-function>ui-sap-business-client--nwbc-
+author_name: Sandra Thimme
+author_profile: https://github.com/sandra36
+---
+
+# Launch native SAP GUI for Windows Applications in the Cloud with SAP Business Client
```
Comment (MichaelCzcz) [inline tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md:11]: Please use title case (->native)

File: tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+<!-- description --> Running SAP Build Work Zone, standard edition inside SAP Business Client you can mix native SAP GUI transactions with state-of-the-art web applications. Therefore, a TCP ping to the remote system has to be performed at the time the application shall be started.
```
Comment (MichaelCzcz) [inline tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md:12]: Please check the style guide for more information on the description.

File: tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+## You will learn
```
Comment (MichaelCzcz) [inline tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md:14]: Don't ask questions in this sections. Just tell the user what he will learn. Please check the style guide for more information.

File: tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+- What is SAP Build Work Zone, standard edition?
+- What are the Integration Prerequisites?
+- What is the difference between direct and tunneled Access?
+- How to run a cloud launchpad within SAP Business Client in 3 steps.
+
+## Prerequisites
+- You have installed SAP Business Client 7.70.
+SAP ONE Support Launchpad ([SAP ONE Support Launchpad](https://launchpad.support.sap.com/#/softwarecenter/template/products/%20_APP=00200682500000001943&_EVENT=DISPHIER&HEADER=Y&FUNCTIONBAR=N&EVENT=TREE&NE=NAVIGATE&ENR=73554900100200012051&V=MAINT&TA=ACTUAL&PAGE=SEARCH/SAP%20BUSINESS%20CLIENT%207.70))
```
Comment (MichaelCzcz) [inline tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md:22]: What is the purpose of this link? If it is part of the first line item, remove the line break.

File: tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+- You have created a **trial account on SAP BTP**:  [Get a Free Account on SAP BTP Trial](hcp-create-trial-account)
```
Comment (MichaelCzcz) [inline tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md:23]: You cannot assume that everybody knows what BTP means. Please write out the full product name when you are mentioning it for the first time and put the abbreviation in the brackets -> SAP Business Technology Platform (SAP BTP).

File: tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+- You are subscribed to the SAP Build Work Zone, standard edition including the Launchpad Admin role: see the following tutorial [Deliver Your First SAP Fiori Launchpad Site] (mission.cp-starter-digitalexp-portal)
+
+See also: [SAP Build Work Zone, standard edition (formerly known as SAP Launchpad Service) on BTP](https://help.sap.com/docs/Launchpad_Service/8c8e1958338140699bd4811b37b82ece/fd79b232967545569d1ae4d8f691016b.html?locale=en-US)
```
Comment (MichaelCzcz) [inline tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md:26]: Format as note

File: tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+## Intro
```
Comment (MichaelCzcz) [inline tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md:28]: This should be a more detailed version of what this tutorial is about. The first paragraph is listing more prerequisites, which should be added to the prerequisites section.

Also, do not use the "royal we" unless you are referring to SAP.

File: tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+
+**Additional Information: Direct Access and Tunneled Access**
+
+Direct access is only possible within the corporate network (or via VPN).
+
+Tunneled Access is possible from the corporate network or from the Internet (for which you need the Cloud Connector). The Cloud Connector is http/https based. The DIAG protocol of SAP GUI for Windows cannot be used. We are therefore using direct access here.
+
+**Procedure**
+The following steps will show you how to launch SAP GUI for Windows applications directly from the FLP@BTP. As an example, we like to create an app/tile in the
```
Comment (MichaelCzcz) [inline tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md:37]: FLP@BTP may cause spelling errors. Please put it in back-ticks.

File: tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+SAP Build Work Zone, standard edition (FLP@BTP) to open an SAP GUI for Windows transaction from a remote system (here U1Y*). To connect, you must create a http destination to this system (step 1 below).
+
+>In general, there are two ways of integrating content into the
+SAP Build Work Zone, standard edition: Manual Integration of Business Content, and integration of content from remote content providers. In this blog, we demonstrate the manual integration of apps. Administrators who need to integrate hundreds of roles will use the federation approach, described at the end of this blog.
+
+
+### Create runtime destinations to access apps
+To launch apps in the
```
Comment (MichaelCzcz) [inline tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md:45]: Please remove line break. You should also rephrase this part to improve readability.

File: tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md [hunk 10]
```diff
@@ -0,0 +1,131 @@
...
+SAP Build Work Zone, standard edition, the service must know where the content/apps are located. It is therefore necessary to create a destination to tell the BTP where your app is running.
+
+1. Logon to the BTP cockpit and open the Destination editor (subaccount=>Overview=> Connectivity=>Destinations)
```
Comment (MichaelCzcz) [inline tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md:48]: the logon, to log on

File: tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md [hunk 11]
```diff
@@ -0,0 +1,131 @@
...
+
+```
+ https://<server>.<domain>.<ext>:<port>.
+
+```
+
+
+   ![Destination configuration](1.png)
+
+ 3. Configure destinations and add additional properties as shown in the screenshot below. For example sap-login-group, message server and instance number, sap-snc-name, can be found in SAP Logon.
```
Comment (MichaelCzcz) [inline tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md:61]: [...]below, for example, sap-login-group,[...]
put sap-snc-name in back-ticks

File: tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md [hunk 12]
```diff
@@ -0,0 +1,131 @@
...
+
+   ![Destination configuration](2.png)
+
+[Link to documentation on SAP Help Portal: Create Runtime Destinations to Access Apps](https://help.sap.com/viewer/8c8e1958338140699bd4811b37b82ece/Cloud/en-US/a57c27ce348241a89d56874bef9a7a07.html)
+
+
+### Make your Apps visible in the Launchpad
```
Comment (MichaelCzcz) [inline tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md:68]: use sentence case

File: tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md [hunk 13]
```diff
@@ -0,0 +1,131 @@
...
+In this step you will:
+
```
Comment (MichaelCzcz) [inline tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md:70]: If the task sequence is relevant, use an ordered list

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
thank you for this new tutorial. I have added some comments. Some of the images should be cropped to show only the relevant information (for example, 12.png). It would also be useful to use filenames that are related to the images' contents or the steps they are used in.
In some comments I am referencing the style guide., that you can find here: https://workzone.one.int.sap/site#workzone-home&/wiki/show/MozDWaPczznL8NKeGEk0y9

Your tutorial is also missing the mandatory validation (-> Test yourself).

**Comment by MichaelCzcz** [comment]
@sandra36 Are you still interested in this pull request?

**Comment by MichaelCzcz** [comment]
@sandra36 I cannot see any commit that you have pushed after my review.

File: tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md (hunk 1 above)
Comment (jmmargo) [reply tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md:11]: This comment has not been addressed.  Please change "native" to "Native" as it should be title case.

File: tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md (hunk 3 above)
Comment (jmmargo) [reply tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md:14]: Please revise like the following:
What SAP Build Work Zone is --> "the definition of SAP Build Work Zone"
What the Prerequisites for the integration of SAP Build Work Zone are --> "the prerequisites for the integration of SAP Build Work Zone"
What the difference between direct and tunneled Access is --> "the difference between direct and tunneled Access"
How to run a cloud launchpad within SAP Business Client in 3 steps.

File: tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md (hunk 5 above)
Comment (jmmargo) [reply tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md:22]: Please follow Michael's suggestion above.

File: tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md (hunk 11 above)
Comment (jmmargo) [reply tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md]: In the markdown, it would look like this: `sap-snc-name`

**Pull request merged**